tests/door43_tools_tests/test_templaters.py

```diff
@@ -48,9 +48,9 @@ def testTemplaterTaComplete(self):
         test_file_path = self.extractZipFiles(test_folder_name)
         success = self.doTemplater('ta', test_file_path)
         self.verifyTaTemplater(success, expect_success, self.out_dir,
-                               ['checking.html', 'intro.html', 'process.html', 'translate.html'])
+                               ['01-intro.html', '02-process.html', '03-translate.html', '04-checking.html'])
```

Comment:
This test templates files from tests/door43_tools/resources/converted_projects/en_ta-complete.zip...that is where it is getting it's pre-templated 01-intro.html file, etc.